src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -50,7 +65,12 @@ public class MsalDistributedTokenCacheAdapter : MsalAbstractTokenCacheProvider
         /// <returns>A <see cref="Task"/> that completes when key removal has completed.</returns>
         protected override async Task RemoveKeyAsync(string cacheKey)
         {
+            var startTicks = Utility.Watch.Elapsed.Ticks;
```

Comment:
nit: here's a little trick you can do achieve the same but with nicer syntax - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/ee169d69820a92fbf9538020592e30a9972ae33d/src/client/Microsoft.Identity.Client/Internal/Logger/DurationLogHelper.cs#L10

syntax would be 

```csharp
using (_logger.LogDuration("[IdWebCache] Do stuff"))
{
     // do stuff
}
```

This would log: 

[IdWebCache] Do stuff started
[IdWebCache] Do stuff finished in 30 ms

Reply:
thanks for the info and link. I took a look, but really we are doing a lot more in the log messages, like logging cache size, cache key, count, and more, not just time, so I think i'll stick to what the MISE team is doing for now and we might revisit this later.

---
In reply to: [586320555](https://github.com/AzureAD/microsoft-identity-web/pull/1037#discussion_r586320555) [](ancestors = 586320555)